### ISSUE TITLE
mdVirtualRepeat: Buggy scrolling near end of list

### ISSUE THREAD

**Issue by aviraldg** [angular/material#5130]
Demo: http://codepen.io/aviraldg/pen/JYyBEL

Try to scroll to the bottom of the list.

Expected: Scrolling stops at end of list.

Actual: 

1. Scroll position "jumps" jarringly when near end of list, `translateY` keeps increasing (despite no actual increase in scroll position)
2. After step 1, scrolling up is also useless and exhibits the same buggy behaviour (apparently until `translateY` returns to an appropriate value)

**Comment by nathanmarks** [comment]
Same thing happening on my end:

https://dl.dropboxusercontent.com/s/i4ll9i4wpzhbjnw/2015-10-13%20at%2010.55%20AM.png

**Comment by RolandHeath** [comment]
This issue is a duplicate of #4169

**Comment by aviraldg** [comment]
Thanks @RolandHeath, hadn't seen that.
